### PR TITLE
Yast2 control center

### PR DESCRIPTION
**Pull request by Zaoliang**
Extend test "yast2 control center"
Please see my reference test at: http://e13.suse.de/tests/1948

### CODE REVIEW

File: tests/yast2_gui/yast2_control_center.pm [hunk 1]
```diff
@@ -23,7 +23,324 @@ sub run() {
...
+    #	start yast2 modules
+    for (1 .. 6) {
+        send_key 'backspace';
+    }
+
+    #	start Add-on Products
+    assert_and_click "yast2_control-center_add-on";
+    assert_screen "yast2_control-center_add-on_installed";
+    send_key "alt-c";
+    assert_screen 'yast2-control-center-ui', 30;
```
Comment (okurz) [inline tests/yast2_gui/yast2_control_center.pm]: please delete the default timeout. Just write `assert_screen '...';`

Comment (Zaoliang) [inline tests/yast2_gui/yast2_control_center.pm]: Okay

File: tests/yast2_gui/yast2_firewall.pm [hunk 2]
```diff
@@ -8,25 +8,62 @@
...
+    #	enter page Custom Rules and check ui
+    assert_and_click("yast2_firewall_custom-rules");
+    send_key "alt-a";
+    assert_screen "yast2_firewall_add-new-allowing-rules";
+    send_key "alt-c";
+    wait_still_screen(2);
+
+    #	Next to finish and exit
+    send_key "alt-n";
+    assert_screen "yast2_firewall_summary", 30;
```
Comment (okurz) [inline tests/yast2_gui/yast2_firewall.pm]: delete default timeout from here

File: tests/yast2_gui/yast2_firewall.pm [hunk 3]
```diff
@@ -8,25 +8,62 @@
...
-    $self->launch_yast2_module_x11($module);
-    assert_screen "yast2-$module-ui", 30;
-    send_key "alt-n";    # Next
-    assert_screen "yast2-$module-summary", 30;
-    send_key "alt-f";    # Finish => Exit
+    my $self = shift;
+
+    $self->launch_yast2_module_x11('firewall');
+
+    assert_screen "yast2-firewall-ui", 30;
```
Comment (okurz) [inline tests/yast2_gui/yast2_firewall.pm]: delete default timeout from here

**Comment by Zaoliang** [comment]
not it should be okay :)

**Pull request merged**